src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Controllers/DecisionController.cs

```diff
@@ -322,9 +322,13 @@ private async Task<XacmlContextResponse> AuthorizeBasedOnDelegations(XacmlContex
 
             foreach (DelegationChange delegation in delegations.Where(d => !d.IsDeleted))
             {
-                XacmlPolicy policy = await _prp.GetPolicyVersionAsync(delegation.BlobStoragePolicyPath, delegation.BlobStorageVersionId);
+                XacmlPolicy delegationPolicy = await _prp.GetPolicyVersionAsync(delegation.BlobStoragePolicyPath, delegation.BlobStorageVersionId);
+                foreach (XacmlObligationExpression obligationExpression in appPolicy.ObligationExpressions)
+                {
+                    delegationPolicy.ObligationExpressions.Add(obligationExpression);
```

Comment:
Agree that all current delegations should just be purged, ie. both the postgre-table and all policyfiles in storage in all environments after this is merged. @altinnadmin, who has access to do this? 

Bit of a tangent, but should we perhaps include som version information in the delegated policies in case we need to make some changes to the stored format?

Reply:
@elsand Check with @TheTechArch or @acn-sbuad in the team slack :) Perhaps they also should do a quick review?